xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -200,43 +199,20 @@ bool CVAAPIContext::CreateContext()
   if (!m_profileCount)
     return false;
 
-  if (!m_attributeCount)
-    CLog::Log(LOGWARNING, "VAAPI - driver did not return anything from vlVaQueryDisplayAttributes");
-
   return true;
 }
 
 void CVAAPIContext::DestroyContext()
 {
-  delete[] m_attributes;
   delete[] m_profiles;
   if (m_display)
     CheckSuccess(vaTerminate(m_display));
 }
 
 void CVAAPIContext::QueryCaps()
 {
-  m_attributeCount = 0;
   m_profileCount = 0;
 
-  int max_attributes = vaMaxNumDisplayAttributes(m_display);
-  m_attributes = new VADisplayAttribute[max_attributes];
```

Comment:
why do you drop this?

Reply:
Same as everything else I dropped previously - not used anywhere. And I don't think it's good to warn about something not being there that's not being used in either case. Or did you want it just for the log output?

Reply:
the log has become useless, right? ok to drop

Reply:
Yeah mesa does not export any display attributes and we actually don't use any